src/defs/accessTokenRepresentation.ts

```diff
@@ -0,0 +1,60 @@
+import AddressClaimSet from './addressClaimSet';
+import {Category} from './resourceServerRepresentation';
+
+export default interface AccessTokenRepresentation {
```

Comment:
I wonder if we want to specify this type, it look as the fields can be quite dynamic. What is this based on exactly? Perhaps a record type of `Record<string, unknown>` would be more appropriate.

Reply:
The AccessToken type definition has these fields listed the docs: https://www.keycloak.org/docs-api/11.0/rest-api/#_accesstoken

`Record<string, unknown>` could also work though - do you think it would make more sense to do it that way?

Reply:
Nice, if they are defined in the docs let's use them as specified there.